src/applications/ask-a-question/config/form.js

```diff
@@ -39,12 +41,17 @@ const submitTransform = (formConfig, form) => {
 };
 
 const formConfig = {
+  rootUrl: manifest.rootUrl,
   urlPrefix: '/',
   submitUrl: `${environment.API_URL}/v0/ask/asks`,
   transformForSubmit: submitTransform,
   trackingPrefix: 'complex-form-',
   confirmation: ConfirmationPage,
   formId: '0873',
+  benefitDescription: {
+    benefitType: '',
+    subText: false,
```

Comment:
This is the kind of question I'm talking about when I say `getBenefitString` and `getSavedFormDescription` are clunky.

I set it to false here to mimic the current (buggy) behavior of displaying nothing in the SiP messages instead of `(0873)`.